getl/fileregistry/delta_diff.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime
+
+from pyspark.sql import DataFrame, functions as F, types as T
+
+from ..block import BlockConfig
+from ..fileregistry import fileregistry_utils as fr_utils
```

Comment:
This is a bit inconsistent with the rest of the repo where we use full paths.

Reply:
:+1: will fix